packages/apollos-data-prayer/package.json

```diff
@@ -39,4 +39,4 @@
   "publishConfig": {
     "access": "public"
   }
-}
\ No newline at end of file
+}
```

Comment:
uhh what github? lol

Reply:
can you get storybook to run on master?

Reply:
that's an error in your node modules, did you `yarn`?

Reply:
PS: This isn't the error image but I'm happy to use this as our convo piece.

Yes. I did yarn.

Reply:
I'm confused. What error are you getting?